src/components/datatable/EarthEngineColumns.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Column } from 'react-virtualized';
+import ColumnHeader from './ColumnHeader';
+import { numberPrecision } from '../../util/numbers';
+import { getPropName, getPrecision } from '../../util/earthEngine';
+
+const EarthEngineColumns = ({ classes, aggregationType, legend, data }) => {
+    const { title, items } = legend;
+
+    if (classes && items) {
+        const valueFormat = numberPrecision(2);
+
+        return items.map(({ id, name }) => (
+            <Column
+                key={id}
+                dataKey={name}
+                label={name}
+                width={100}
+                className="right"
+                headerRenderer={props => (
+                    <ColumnHeader type="number" {...props} />
+                )}
+                cellRenderer={d =>
+                    d.cellData !== undefined ? valueFormat(d.cellData) : ''
+                }
+            />
+        ));
+    } else if (Array.isArray(aggregationType)) {
```

Comment:
Should `null` be returned instead of `[]` if `aggregationType` has no elements?
```suggestion
    } else if (Array.isArray(aggregationType) && aggregationType.length) {
```

Reply:
Fixed.